src/accessibility/LandmarkNavigation.ts

```diff
@@ -18,6 +18,12 @@ import { TimelineRenderingType } from "../contexts/RoomContext";
 import { Action } from "../dispatcher/actions";
 import defaultDispatcher from "../dispatcher/dispatcher";
 
+/**
+ * In future, browsers will support focusVisible option.
+ * See https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus#focusvisible
+ */
+type ExperimentalFocusOptions = FocusOptions & { focusVisible: true };
```

Comment:
Can you add this type in `global.d.ts` instead to avoid needing to type cast and to get the benefit in all files?

Reply:
done, https://github.com/matrix-org/matrix-react-sdk/pull/12190/commits/d1085b7b8ef877d1481788b849c22ce805461b33

Reply:
It might need to be optional `focusVisible?: boolean` but CI will show